blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/FlowNodeWrapper.java

```diff
@@ -203,6 +204,9 @@ public void setBlockErrorAction(ErrorAction blockErrorAction) {
      * Filters by class to mimic Item.getActions(class).
      */
     public <T extends Action> Collection<T> getPipelineActions(Class<T> clazz) {
+        if(pipelineActions == null){
```

Comment:
Fixes NPE happening in cases where there are no actions added.

Reply:
Cheers!